### ISSUE TITLE
gallbladder polyposis

### ISSUE THREAD

**Issue by nklsbckmnn** [obophenotype/human-phenotype-ontology#11065]
**Preferred term label:**
gallbladder polyposis

**Synonyms**

**Definition (free text, please give PubMed ID)**
Multiple polyps arising from the gallbladder mucosal wall.

**Parent term (use hpo.jax.org/app)**
gallbladder polyp

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0009-0009-6791-365X

**Comment by pnrobinson** [comment]
@nklsbckmnn 
Adding new term: 
- ID: HP:6001261
- label: Gallbladder polyposis
- parents: [Abnormal gallbladder morphology (HP:0012437)](https://hpo.jax.org/app/browse/term/HP:0012437)
- pmid: PMID:29262128
- def: Multiple polyps arising from the gallbladder mucosal wall.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11065
- synonyms:
